src/Fizzics/model.js

```diff
@@ -60,7 +60,7 @@ function _addPropsForIndex(props, index) {
         _propFlags, 0.0, 0.2, 0.1);
     props[`friction-${index}`] = GObject.ParamSpec.double(
         `friction-${index}`, `friction-${index}`, '',
-        _propFlags, 0.0, 18.0, 9.0);
+        _propFlags, 0.0, 100.0, 5.0);
```

Comment:
Since you haven't (yet) changed the max/min definitions on the client side, I just want to double check that you intention is to set:

max friction: 100.0
min friction: 0.0
default friction: 5.0

This is what both the model.js and .ui are doing.

Reply:
That's correct.